Mixpanel/SessionMetadata.m

```diff
@@ -0,0 +1,52 @@
+//
+//  SessionMetadata.m
+//  Mixpanel
+//
+//  Created by Yarden Eitan on 10/27/17.
+//  Copyright © 2017 Mixpanel. All rights reserved.
+//
+
+#import "SessionMetadata.h"
+
+@implementation SessionMetadata
+
+- (instancetype)initWithDispatchQueue:(dispatch_queue_t)queue {
+    self = [super init];
+    if (self) {
+        self.eventsCounter = 0;
+        self.peopleCounter = 0;
+        self.sessionID = 0;
+        self.sessionStartEpoch = 0;
```

Comment:
i think we should init `sessionStartEpoch` to `(uint64_t)[[NSDate date] timeIntervalSince1970];`